dockerfiles/models-gpu/Dockerfile

```diff
@@ -149,7 +149,7 @@ RUN python -c "from matplotlib.font_manager import _rebuild; _rebuild();"
 
 RUN conda clean -aqy
 
-RUN python -m pip install git+https://github.com/${REPO}/mlrun.git@${MLRUN_TAG}
+RUN python -m pip install git+https://github.com/${MLRUN_GITHUB_REPO}/mlrun.git@${MLRUN_PACKAGE_TAG}
 
 RUN python -m pip install --ignore-installed --no-cache-dir \
```

Comment:
looks like same requirement as `base`, have a common requirement.txt to unite them all and install when needed

Reply:
@yjb-ds this is for you
@liranbg see above comment about requirements,